### ISSUE TITLE
Main content for the 'Home' page

### ISSUE THREAD

**Issue by michaella23** [ScrimbaBootcamp/ScrimbaBootcamp2024#9]
What should this be? Right now, all there may be is an `<h1>`. What kind of content should be on the home page of this open-source project? Let's discuss. You can suggest something you would like to add when you request assignment to this issue.

**Comment by michaella23** [comment]
Thanks @AnBC22! I just started a discussion in the [#collaborative-work channel](https://discord.com/channels/684009642984341525/1118337850207907910). let's see if anyone has ideas!

**Comment by AnBC22** [comment]
Yes, I saw it. Awesome!

**Comment by AnBC22** [comment]
@michaella23 In this case we are ready to start? Do I do this about the news and articles?

**Comment by michaella23** [comment]
@AnBC22, yes let's go for it Andrés! I was scouting out news APIs, and you might want to check out newsdata.io. You don't have to use that! Just an idea. Thanks for taking this on!

**Comment by AnBC22** [comment]
Awesome Micha! Thanks for the suggestion, I'll use that and let you know when I'm done
